README.md

```diff
@@ -200,9 +200,14 @@ sudo apt install make gcc ripgrep unzip git neovim
 ```
 sudo apt update
 sudo apt install make gcc ripgrep unzip git
-echo "deb https://deb.debian.org/debian unstable main" | sudo tee -a /etc/apt/sources.list
-sudo apt update
-sudo apt install -t unstable neovim
+
+# Now we install nvim
+curl -LO https://github.com/neovim/neovim/releases/latest/download/nvim-linux64.tar.gz
+sudo rm -rf /opt/nvim
+sudo tar -C /opt -xzf nvim-linux64.tar.gz
```

Comment:
when testing this I noticed that `/opt/nvim-linux64` was inaccessible to my user because of umask 027 and the `nvim-linux64` itself is not in the archive so tar would not set it's permissions, so perhaps this needs to be added:
```
sudo mkdir -p /opt/nvim-linux64
sudo chmod a+rx /opt/nvim-linux64
```

Reply:
Or just put it in a subshell `(umask 022; sudo tar...)` , i mean it's not a hard workaround but you're going to be changing every guide online for it.

if you set `0277` you don't want things installed as root run by non-root, that's what you're saying. some people like that. if so, you could install it to `~/opt/` and symlink to `~/.bin/` rather thank `/usr/local/bin`.

Reply:
That's not what I'm saying, a user having a umask  027 intention is that the user's files are not readable by other users by default. I think that's not an uncommon setting. That doesn't translate to the intention of root installed packages not being able to be used by users. If the `/opt/nvim-linux64` was a part of the tar package this would not be an issue because tar will preserve permissions but it is not, only the subdirs are part of the tar package.
Anyway, this is just a suggestion to make it less error prone for a wider audience, feel free to ignore it and resolve this comment if you think the current behaviour is preferred.